app/build.gradle

```diff
@@ -240,7 +240,7 @@ dependencies {
     kapt "com.google.dagger:dagger-compiler:$dagger_version"
     implementation 'com.jakewharton:process-phoenix:3.0.0'
     implementation "com.jakewharton.timber:timber:5.0.1"
-    implementation 'com.stripe:stripe-android:20.47.3'
```

Comment:
[21.7.1](https://mvnrepository.com/artifact/com.stripe/stripe-android/21.7.1) is the latest, requires kotlin version upgrades